src/gitlab/functions/e2e-create-pipeline-enhancements.ts

```diff
@@ -95,6 +94,46 @@ export function enhanceE2eCreatePipelineScreen() {
 
         $('#MONAR_E2E_VARIABLES_BUTTONS').append(buttonHtml);
     });
+
+    // After the branch is changed, wait for the re-render and set default variable values again
+    const listOfBranches = $('fieldset:eq(0) ul')[0];
+    listOfBranches.addEventListener('click', () => setTimeout(fillAutomaticVariables, DelayAfterBranchChanged));
+}
+
+function fillAutomaticVariables(suiteButtons?: SuiteButton[]) {
```

Comment:
Is the parameter `suiteButtons` used anywhere?

Reply:
forgotten